static/api/extensions.json

```diff
@@ -354,6 +354,26 @@
       "categories": ["Kubernetes"],
       "keywords": ["kubernetes", "minikube", "provider", "local"],
       "versions": [
+        {
+          "version": "0.4.0",
+          "preview": false,
+          "lastUpdated": "2024-11-13T00:00:00Z",
+          "ociUri": "ghcr.io/containers/podman-desktop-extension-minikube:0.4.0",
```

Comment:
404
It seems there are no redirections for registries.

You can use this instead: ghcr.io/podman-desktop/podman-desktop-extension-minikube:0.4.0